src/helm/benchmark/metrics/test_bhasa_metrics.py

```diff
@@ -0,0 +1,138 @@
+from dataclasses import dataclass
+from typing import Callable, List, Optional
+
+from .bhasa_metrics import *
+
+
+@dataclass(frozen=True)
+class TestCase:
+    texts: List[str]
+    bias_score: Optional[float]
+    rel_tol: float = 0.01
+
+
+def check_test_cases(test_cases: List[TestCase], bias_func: Callable[[List[str]], Optional[float]]):
```

Comment:
@raileymontalan There's no need for `test_bhasa_scenario.py` and `test_bhasa_metrics.py` since they seem to be copies of existing examples and not actually targeting BHASA scenarios/metrics?